### PR TITLE
feat(connectors): add logo for connectors

### PR DESCRIPTION
**Pull request by darcyYe**
## Summary

Add connectors' logo.

## Linear Issue Reference

LOG-1838

## Testing

Passed UTs.
Before:
image
After:
image

### CODE REVIEW

**Review by IceHe — APPROVED** [review]
LGTM

**Comment by darcyYe** [comment]
image
image
etc.

File: packages/connector-alipay-native/src/constant.ts [hunk 1]
```diff
@@ -29,7 +29,7 @@ export const defaultMetadata: ConnectorMetadata = {
     en: 'Alipay',
     'zh-CN': '支付宝',
   },
-  logo: './logo.png',
+  logo: 'https://gist.githubusercontent.com/darcyYe/31bc893a0a305dc43cf831bf0b14f0fc/raw/faf985d3fbeed88180b8f3cb709892320d66ae45/alipay.svg',
```
Comment (demonzoo) [inline packages/connector-alipay-native/src/constant.ts:32]: I'm curious where we can find these svg source files. Are they uploaded to one of your private repos?

Comment (darcyYe) [reply packages/connector-alipay-native/src/constant.ts:32]: Currently, they are on my GitHub gist but are publicly accessible. Due to the network accessibility issue in some areas, @wangsijie will put them on some other server, here is the [issue](https://linear.app/silverhand/issue/LOG-2594/serve-connectors-logo).

Comment (demonzoo) [reply packages/connector-alipay-native/src/constant.ts:32]: ok

**Review by demonzoo — APPROVED** [review]

**Review by wangsijie — APPROVED** [review]

**Pull request merged**